Fix up for bz 595413

Mistake in the man page for blkrecord:
documented option --input-base is unsupported,
supported option --max-bunch-time is undocumented.

Signed-off-by: Edward Shishkin <edward@redhat.com>
---
--- blktrace-1.0.1.orig/doc/btrecord.8
+++ blktrace-1.0.1/doc/btrecord.8
@@ -107,7 +107,7 @@
 
 \-m <\fInanoseconds\fR>
 .br
-\-\-input\-base=<\fInanoseconds\fR>
+\-\-max\-bunch\-time=<\fInanoseconds\fR>
 .RS
 The \fI\-m\fR option requires a single parameter which specifies an
 amount of time (in nanoseconds) to include in any one bunch of IOs that
